src/Sulu/Component/Security/Authorization/AccessControl/AccessControlManagerInterface.php

```diff
@@ -26,7 +26,7 @@ interface AccessControlManagerInterface
      * @param string $identifier The identifier of the protected object
      * @param mixed[] $permissions
      */
-    public function setPermissions($type, $identifier, $permissions);
+    public function setPermissions($type, $identifier, $permissions/*, bool $inherit = false */);
```

Comment:
I think in this interface I would not care too much about BC, or do you know about anybody who has extended this class in a way that this would make it break? If we don't do it this way it will be way easier to maintain.

Reply:
No does not know any project which did extend this class. So I think we could change it.